minikube/bootstrap.sh

```diff
@@ -116,147 +116,106 @@ waitDashboardIsDeployed() {
   set -e
 }
 
-minikubeVer(){
-LATEST_VER=`curl -s https://github.com/kubernetes/minikube/releases/latest| sed -e 's/.*v\(.*\)".*/\1/'`
-
-LATEST_VER_SHA=`curl -s -L https://github.com/kubernetes/minikube/releases/download/v$VER/minikube-linux-$ARCH.sha256 -o \
-    /tmp/minikube.sha && cat /tmp/minikube.sha`
-
-CURRENT_VER=`sha256sum /usr/local/bin/minikube | awk '{print $1}'`
-
-if ! [[ "$LATEST_VER_SHA" == "CURRENT_VER" ]]; then
-echo "curl -s -LO https://github.com/kubernetes/minikube/releases/download/v$VER/minikube-linux-$ARCH"
-fi
+#helmLinuxUpgrade check new version and upgrade helm using version number
+helmLinuxUpgrade() {
+  unset LATEST_VER CURRENT_VER SHA CHECK_VER
+  LATEST_VER=`curl -s https://github.com/kubernetes/helm/releases/latest | sed -e 's/.*v\(.*\)".*/\1/'`
+  CURRENT_VER=`helm version 2>/dev/null | sed -n '/.*v/s///p' | grep -o '^[^"]*' | head -n 1`
+  if ! [[ "$CURRENT_VER" == "$LATEST_VER" ]]; then
```

Comment:
Nope, it's better to compare file hashes. Because files may change even for the same version.

Reply:
Oh, I see it's [here](#diff-904c401c51ca9723b75a72547bdb4f63R133).

Why bother to implement 2 funcs? Leave only variant with hashes.

Reply:
@tyranron I wrote this variant as alternative and want to ask you what is better. I`ll use variant with hashes.